package.json

```diff
@@ -34,9 +33,6 @@
     "xstream": "*"
   },
   "devDependencies": {
-    "@cycle/run": "3.1.0",
-    "typescript": "^2.3.4",
-    "typings": "^2.1.1",
-    "xstream": "^10.8.0"
```

Comment:
Can you please take some time, verify this, and get back? Would be very useful for us to know what is really needed and what isn't. Thank you for taking the time. :)